targets/TARGET_STM/README.md

```diff
@@ -458,7 +458,39 @@ https://github.com/ARMmbed/mbed-os/blob/master/connectivity/drivers/emac/TARGET_
 Option is also to define your own `HAL_ETH_MspInit` function,
 you then have to add **USE_USER_DEFINED_HAL_ETH_MSPINIT** macro.
 
+To change the default MAC address in STM32,
+If we have the function mbed_otp_mac_address() in the user application,the default ethernet address
+can be changed.
+Because as this is defined as weak in mbed-os/connectivity/drivers/emac/TARGET_STM/stm32xx_emac.cpp
+```
+__weak uint8_t mbed_otp_mac_address(char *mac).
```

Comment:
as this is for Mbed OS, lets use `MBED_WEAK` instead

```suggestion
#include "platform/mbed_toolchain.h"
MBED_WEAK uint8_t mbed_otp_mac_address(char *mac).
```

Reply:
@0xc0170 Suggested changes are updated.